general_superstaq/check/check_utils.py

```diff
@@ -178,64 +164,105 @@ def _revision_exists(revision: str) -> bool:
     )
 
 
-def get_test_files(*files: str, exclude: Union[str, Iterable[str]] = "", silent: bool) -> List[str]:
+def get_test_files(
+    *files: str, exclude: Union[str, Iterable[str], None] = None, silent: bool
+) -> List[str]:
     """
     For the given files, identify all associated test files (i.e. files with the same name, but
     with a "_test.py" suffix).
     """
-    should_include = inclusion_filter(exclude)
 
     test_files = set()
     for file in files:
-        if file.endswith("_test.py"):
+        if file.split("::")[0].endswith("_test.py"):
             test_files.add(file)
-
         else:
             test_file = re.sub(r"\.py$", "_test.py", file)
-            test_file_exists = os.path.isfile(os.path.join(root_dir, test_file))
-            if test_file_exists and should_include(test_file):
+            if os.path.isfile(os.path.join(root_dir, test_file)):
                 test_files.add(test_file)
             elif not silent:
                 print(warning(f"WARNING: no test file found for {file}"))
 
-    return list(test_files)
+    if exclude is not None:
+        exclude = [exclude] if isinstance(exclude, str) else list(exclude)
```

Comment:
nit: `exclude_files` already accepts an `Iterable`, so this can just be
```suggestion
        exclude = [exclude] if isinstance(exclude, str) else exclude
```

Reply:
The same is true in a few more places in this PR